src/assets/CollectiblesController.ts

```diff
@@ -201,13 +201,12 @@ export class CollectiblesController extends BaseController<
   private updateNestedCollectibleState(
     newCollection: Collectible[] | CollectibleContract[],
     baseStateKey: 'allCollectibles' | 'allCollectibleContracts',
-    passedConfig?: { selectedAddress: string; chainId: string },
+    passedConfig?: AccountParams,
   ) {
     // We want to use the passedSelectedAddress and passedChainId when defined and not null
     // these values are passed through the collectible detection flow, meaning they may not
     // match as the currently configured values (which may be stale for this update)
-    const address =
-      passedConfig?.selectedAddress ?? this.config.selectedAddress;
+    const address = passedConfig?.userAddress ?? this.config.selectedAddress;
     const chain = passedConfig?.chainId ?? this.config.chainId;
```

Comment:
[nit] I think you can simplify this as follows:

``` suggestion
    { userAddress: address, chainId: chain }: AccountParams | undefined = {
      userAddress: this.config.selectedAddress,
      chainId: this.config.chainId
    }
  ) {
```

Reply:
updated here: https://github.com/MetaMask/controllers/pull/672/commits/0dcc89542336640a1f8dccb2cd9bca82b4c1dd03